saga-spring/src/test/java/io/servicecomb/saga/spring/SagaControllerTest.java

```diff
@@ -75,4 +88,76 @@ public void retrievesAllEvents() throws Exception {
         .andExpect(status().isOk())
         .andExpect(content().json(objectMapper.writeValueAsString(events)));
   }
+
+  @Test
+  public void queryRequests() throws Exception {
+    long id = 0L;
+
+    String sagaId1 = "xxxx";
+    SagaEventEntity saga1StartEvent = new SagaEventEntity(++id, sagaId1, new Date().getTime() + id * 50,
+        SagaStartedEvent.class.getSimpleName(), "{}");
+    SagaEventEntity saga1EndEvent = new SagaEventEntity(++id, sagaId1, new Date().getTime() + id * 50,
```

Comment:
preferably to use OffsetDateTime instead of new Date().

```
now = OffsetDateTime.now();
now.plusMinutes(1);
Date.from(now.toInstance());
```

Reply:
Done